sample/index.ts

```diff
@@ -26,9 +23,7 @@ import "redis-sub";
 // Verify that patches are applied
 console.dir((<any>channel).getPatchesObject());
 
-import {IConsoleData} from 'console-pub';
-
-declare var process;
+import {IConsoleData} from '../src/diagnostic-channel-publishers/.dist/src/console.pub';

```

Comment:
Instead of a filesystem relative path, what about

```ts
import {console as consolePub} from "diagnostic-channel-publishers";
import IConsoleData = consolePub.IConsoleData;
```